catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitQueryRequestImpl.java

```diff
@@ -241,11 +241,14 @@ private void notifyCallbacks() {
 
         try {
             QueryResultsCallable queryCallable = new QueryResultsCallable(catalogQueryRequest);
-            results.addAll(guestSubject.execute(queryCallable));
+            results.addAll(NsiliEndpoint.getGuestSubject().execute(queryCallable));
 
         } catch (ExecutionException e) {
             LOGGER.warn("Unable to query catalog {}", e);
             LOGGER.debug("Catalog query exception details", e);
+        } catch (SecurityServiceException e) {
```

Comment:
The warn should just print the toString where the debug should be the stack trace.